docs/source/conf.py

```diff
@@ -201,14 +221,25 @@
     'cv2',
     'cython',
     'keras-preprocessing',
+    'deepcell_tracking',
+    'keras_applications',
+    'matplotlib'
 ]
 
+sys.modules['deepcell.utils.compute_overlap'] = mock.Mock()
+
+# Disable nbsphinx extension from running notebooks
+nbsphinx_execute = 'never'
+exclude_patterns = ['_build', '**.ipynb_checkpoints']
+
 # -- Options for intersphinx extension ---------------------------------------
 
 intersphinx_mapping = {
-    'kiosk': ('https://deepcell-kiosk.readthedocs.io/en/latest/', None),
     'python': ('https://docs.python.org/3.7', None),
     'numpy': ('http://docs.scipy.org/doc/numpy/', None),
+    'kiosk': ('https://deepcell-kiosk.readthedocs.io/en/{}/'.format(rtd_version), None),
+    'kiosk-redis-consumer': ('https://deepcell-kiosk.readthedocs.io/projects/kiosk-redis-consumer/en/{}/'.format(rtd_version), None),
+    'kiosk-frontend': ('https://deepcell-kiosk.readthedocs.io/projects/kiosk-frontend/en/{}/'.format(rtd_version), None)
```

Comment:
@willgraf Is it possible to disable the. line too long errors here and in a couple other places in this file?

Reply:
You can use `# pylint: disable=line-too-long` (I think that is the name of the error).

You can also just have the string break into multiple lines if it is inside parentheses, like:

```python
("This is the beginning of the string"
 "This is a continued part of the same string")
```

Reply:
Not sure if this was ever resolved, but you should just add `docs` to the `pytest.ini` file:

```
norecursedirs= build, .ipynb_checkpoints, docs
```